packages/mobile/e2e/src/utils/komenci.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios').default
+
+export async function checkKomenci() {
+  try {
+    const response = await axios.get('https://staging-komenci.azurefd.net/v1/ready')
```

Comment:
Maybe we can use a constant for this?

Reply:
Moved to constants in 3c0a3c71cce58d81df741707189f3947ac560141.